src/test/java/net/imagej/scripting/TestScript.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package net.imagej.scripting;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import javax.script.ScriptException;
+import net.imagej.Dataset;
+import org.junit.Test;
+import org.scijava.script.ScriptModule;
+
+/**
+ *
+ * @author Hadrien Mary
+ */
+public class TestScript extends AbstractScriptTest {
```

Comment:
In line with the current ImageJ conventions, and to be consistent with `AbstractScriptTest`, the class name should be `ScriptTest` or maybe some more specific `ImageJ2ScriptTest`, so there can also be `DeconvolutionScriptTest` etc.

IIRC, throughout the ImageJ universe (e.g. imagej-ops), the convention is to have an interface (i.e. `interface ScriptTest`) that is _implemented_ by an abstract class that is then _extended_ by a default implementation or several specific ones.